Assets/AnkrSDK/Scripts/Core/Data/EventFilterData.cs

```diff
@@ -4,10 +4,22 @@ namespace AnkrSDK.Core.Data
 {
 	public class EventFilterData
 	{
-		public object[] filterTopic1;
-		public object[] filterTopic2;
-		public object[] filterTopic3;
-		public BlockParameter fromBlock;
-		public BlockParameter toBlock;
+		public object[] filterTopic1 { get; set; }
+		public object[] filterTopic2 { get; set; }
+		public object[] filterTopic3 { get; set; }
+		public BlockParameter fromBlock { get; set; }
+		public BlockParameter toBlock { get; set; }
+
+		public bool TopicsIsFilled()
+		{
+			return filterTopic1 != null || filterTopic2 != null || filterTopic3 != null;
+		}
+	}
+	
+	public class EventFilterRequest<TEvDTO>
```

Comment:
You should not have 2 different classes in one file. Move this into its own .cs file